src/ResourceManager/Advisor/Commands.Advisor/help/Disable-AzureRmAdvisorRecommendation.md

```diff
@@ -0,0 +1,175 @@
+---
+external help file: Microsoft.Azure.Commands.Advisor.dll-Help.xml
+Module Name: Az.Advisor
+online version:
+schema: 2.0.0
+---
+
+# Disable-AzureRmAdvisorRecommendation
+
+## SYNOPSIS
+Disable an Azure Advisor recommendation.
+
+## SYNTAX
+
+### IdParameterSet (Default)
+```
+Disable-AzureRmAdvisorRecommendation [-ResourceId] <String> [[-Days] <String>]
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### NameParameterSet
+```
+Disable-AzureRmAdvisorRecommendation [[-Days] <String>] [-RecommendationName] <String>
+ [-DefaultProfile <IAzureContextContainer>] [<CommonParameters>]
+```
+
+### InputObjectParameterSet
+```
+Disable-AzureRmAdvisorRecommendation [[-Days] <String>]
+ [-InputObject] <PsAzureAdvisorResourceRecommendationBase> [-DefaultProfile <IAzureContextContainer>]
+ [<CommonParameters>]
+```
+
+## DESCRIPTION
+Create suppression for recomendation, this enables a particular recommendation to be postponed for a specific duration or infinitely.
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> Disable-AzureRMAdvisorRecommendation -Name "f380a3a8-9d18-cfad-78e0-55762c72a178"
+
+SuppressionId : d1f70547-0e72-db29-443e-c1164d5d4377
+Ttl           : -1
+Id            : /subscriptions/{user_subscription}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cache/Redis/xyz/providers/Microsoft.Advisor/recommendations
+                /{recommendation_id}/suppressions/HardCodedSupressionName
+Name          : HardCodedSupressionName
+Type          : Microsoft.Advisor/suppressions
+```
+
+Create a suppression for the given recomendation name with a default-SuppressionName and default days as -1.
+
+### Example 3
+```powershell
+PS C:\> Disable-AzureRMAdvisorRecommendation -ResourceId "/subscriptions/{user_subscription}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cache/Redis/xyz/providers/Microsoft.Advisor/recommendations/{}" -Days 12
+
+SuppressionId : 7d1f0547-0e72-db29-443e-c1164d5d4377
+Ttl           : 12.00:00:00
+Id            : /subscriptions/{user_subscription}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cache/Redis/xyz/providers/Microsoft.Advisor/recommendations
+                /{recommendation_id}/suppressions/HardCodedSupressionName
+Name          : HardCodedSupressionName
+Type          : Microsoft.Advisor/suppressions
+```
+
+A suppression is created for the given recommendation-Id
+
+### Example 4
+```powershell
+PS C:\>  Get-AzureRMAdvisorRecommendation -Id "/subscriptions/658c8950-e79d-4704-a903-1df66ba90258/resourceGroups/AzExpertStg/providers/Microsoft.Cache/Redis/azacache/providers/Microsoft.Advisor/recommendations/f380a3a8-9d18-cfad-78e0-55762c72a178" | Disable-A
```

Comment:
please remove subscription info or anything else that is not generic.  Apply everywhere.